package.json

```diff
@@ -61,7 +61,7 @@
     "pre-push": "yarn prettify && yarn lint && yarn localize && yarn jest",
     "cy:verify": "cypress verify",
     "cy:e2e:open": "CYPRESS_E2E=true cypress open",
-    "cy:e2e:run": "CYPRESS_E2E=true cypress run"
+    "cy:e2e:run": "CYPRESS_E2E=true cypress run --browser chrome"
```

Comment:
Once we updated to latest packages on project-monitoring we no longer needed to use the chrome browser, the default electron worked fine. Just as a note, probably there is nothing wrong with using chrome, now cypress can record videos also with Chrome.

Reply:
I found that with electron the cookies are not separated by port, so when performing synchronizations between two instances on localhost the JSESSION is overwritten and the tests do not pass.

Right now we don't have two dockers yet but once we set them up, we will need to run them on chrome.